main/2025/CG-10.md

```diff
@@ -0,0 +1,75 @@
+![WebAssembly logo](/images/WebAssembly.png)
+
+## Table of Contents
+
+* [Agenda for the October meeting](#agenda-for-the-October-meeting-of-webassemblys-community-group)
+
+   * [Research day](#research-day)
+   * [Registration](#registration)
+   * [Logistics](#logistics)
+
+* [Meeting notes](#meeting-notes)
+
+
+## Agenda for the October meeting of WebAssembly's Community Group
+
+- **Host**: Google, Munich, Germany
+- **Dates**: Tuesday-Wednesday, October 28-29, 2025
+- **Times**: 9:00AM - 5:00PM PT
```

Comment:
```suggestion
- **Times**: 9:00AM - 5:00PM CET
```

Reply:
Or I suppose if you wanted to go to full local style you could do 09:00 - 17:30.
Also, looks like the US is still in daylight time here so we're hitting one of those weeks where there's more overlap.